content/doc/book/security/csrf-protection.adoc

```diff
@@ -50,6 +50,7 @@ The plugin:strict-crumb-issuer[Strict Crumb Issuer] provides an alternative crum
 ## Working with Scripted Clients
 
 Requests sent using the `POST` method are subject to CSRF protection in Jenkins and generally need to provide a crumb.
+This applies to scripted clients that use anonymous authentication.
 This also applies to scripted clients that **authenticate using username and password**.
```

Comment:
```suggestion
This applies to scripted clients that use anonymous authentication
as well as scripted clients that **authenticate using username and password**.
```

Reply:
What if we added a WARNING: block here?  This allows us to provide the information but clarifies that this should not be done lightly.  @daniel-beck , if you could articulate the risk, I will draft the prose.  I'm thinking of something like:

WARNING: Running scripted clients with anonymous authentication
should be avoided because it can compromise the security of your installation.
<Need brief explanation of the security issue>.
Here, we are running a pre-production setup
(meaning that Jenkins is started with -Djenkins.install.runSetupWizard=false
and configured entirely using configuration-as-code HTTP POSTs)
so the initial access is anonymous.